### ISSUE TITLE
Typescript Fix

### ISSUE THREAD

**Issue by cosmascj** [nysamnang/react-native-raw-bottom-sheet#164]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-raw-bottom-sheet@2.2.0` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-raw-bottom-sheet/index.d.ts b/node_modules/react-native-raw-bottom-sheet/index.d.ts
index 31a5259..c43f9cc 100644
--- a/node_modules/react-native-raw-bottom-sheet/index.d.ts
+++ b/node_modules/react-native-raw-bottom-sheet/index.d.ts
@@ -20,6 +20,8 @@ declare module "react-native-raw-bottom-sheet" {
       draggableIcon?: StyleProp<ViewStyle>;
     };
     keyboardAvoidingViewEnabled?: boolean;
+    children: JSX.Element | JSX.Element[]
+
   };

**Comment by liyahemanshu** [comment]
Thanks this fixed the type issue

**Comment by cristo-rabani** [comment]
so, why it's not added?

**Comment by nysamnang** [comment]
Merged #146
